### PR TITLE
Chapters 11 and 12 merged to "Packaging Projects"

### PR DESCRIPTION
**Pull request by bast**

### CODE REVIEW

File: chapter-11/recipe-02/cxx-example/CMakeLists.txt [hunk 1]
```diff
@@ -1,25 +1,85 @@
-cmake_minimum_required(VERSION 3.6 FATAL_ERROR)
+# define minimum cmake version
+cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
 
-project(recipe-02 LANGUAGES CXX)
+# project name and supported language
+project(recipe-01 LANGUAGES CXX)
```
Comment (robertodr) [inline chapter-11/recipe-02/cxx-example/CMakeLists.txt]: Should be `recipe-03` (see comment on general `README.md`)

File: chapter-11/recipe-04/cxx-example/CMakeLists.txt [hunk 2]
```diff
@@ -1,25 +1,28 @@
-cmake_minimum_required(VERSION 3.6 FATAL_ERROR)
+# define minimum cmake version
+cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
 
-project(recipe-04 LANGUAGES CXX)
+# project name and supported language
+project(recipe-03 LANGUAGES CXX)
```
Comment (robertodr) [inline chapter-11/recipe-04/cxx-example/CMakeLists.txt]: Should be `recipe-05`

File: chapter-11/recipe-05/cxx-example/CMakeLists.txt [hunk 3]
```diff
@@ -1,25 +1,55 @@
-cmake_minimum_required(VERSION 3.6 FATAL_ERROR)
+# define minimum cmake version
+cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
 
-project(recipe-05 LANGUAGES CXX)
+# project name and supported language
+project(recipe-04 LANGUAGES CXX)
```
Comment (robertodr) [inline chapter-11/recipe-05/cxx-example/CMakeLists.txt]: Should be `recipe-06`

File: chapter-11/README.md [hunk 4]
```diff
@@ -1,7 +1,11 @@
...
 - [Generating source archives](recipe-01/README.md)
-- [Generating binary archives](recipe-02/README.md)
-- [Generating DEB packages](recipe-03/README.md)
-- [Generating RPM packages](recipe-04/README.md)
-- [Generating an NSIS installer for Windows](recipe-05/README.md)
+- [Generating binary archives](recipe-01b/README.md)
+- [Generating DEB packages](recipe-01c/README.md)
+- [Generating RPM packages](recipe-01d/README.md)
+- [Generating an NSIS installer for Windows](recipe-01e/README.md)
+- [Distributing a C++/Python project built with CMake/pybind11 via PyPI](recipe-02/README.md)
```
Comment (robertodr) [inline chapter-11/README.md]: Accordingly, these will be bumped one number up: 2->3, 3->4, 4->5, 5->6

**Comment by bast** [comment]
So the problem is wrong numbering inside `CMakeLists.txt` in chapter 11, right? Or is there more?

**Comment by robertodr** [comment]
Yes, and directory names for recipes and in the `README.md`(but that's autogenerated).

File: chapter-11/README.md (hunk 4 above)
Comment (robertodr) [reply chapter-11/README.md]: I'll wait for Travis to finish and then merge.

Comment (bast) [reply chapter-11/README.md]: Once CI is through and you are happy, please approve but do not merge, I will then merge and want to keep it as reminder to myself to also adjust TC and internal document numbering.

**Pull request merged**